src/components/SelectedBeer.js

```diff
@@ -0,0 +1,40 @@
+import React, {useState, useEffect} from 'react'
+import axios from 'axios'
+import Header from './Header' 
+
+function SelectedBeer(props) {
+     const [selectedBeer, setBeerState] = useState({})
+     const beerInfo = props.match.params.beerId
+
+     useEffect(() => {
+        axios.get(`https://ih-beers-api2.herokuapp.com/beers/${beerInfo}`)
+        .then(selectedBeer => {
+            setBeerState(selectedBeer.data)
+        })
+        .catch((error)=> "Error while fetching the beer from API")
+       })
```

Comment:
To indicate that this will run only once, you should add an empty dependency array here.
```
useEffect(() => {
        axios.get(`https://ih-beers-api2.herokuapp.com/beers/${beerInfo}`)
        .then(selectedBeer => {
            setBeerState(selectedBeer.data)
        })
        .catch((error)=> "Error while fetching the beer from API")
       }, [])